### ISSUE TITLE
Does Skywalking support istio 1.10?

### ISSUE THREAD

**Issue by zackzhangkai** [apache/skywalking#7241]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [x] Question or discussion
- [ ] Bug
- [ ] Requirement
- [ ] Feature or performance improvement

___
### Question
- What do you want to know?
Does Skywalking support istio1.10?

I want to collect the tracing of request with skywalking. But in istio1.10 the `adapter` crd has been removed.

https://github.com/apache/skywalking-kubernetes/blob/master/chart/skywalking/templates/istio-adapter/adapter.yaml#L17

```
{{- if .Values.oap.istio.adapter.enabled }}
apiVersion: "config.istio.io/v1alpha2"
kind: adapter
metadata:
  name: skywalking-adapter
spec:
  description: "skywaling-adapter"
  session_based: false
  templates:
    - metric
{{- end}}
```

**Comment by wu-sheng** [comment]
First of all, this was Istio's legacy adaptor, which could and is planned to be removed for a long time.
Second, this is not for tracing, this is for metrics, topology, etc.

The highest value of service mesh observability is that don't need to install an agent, which doesn't include tracing. If you need tracing, Envoy and latest(1.11) have SkyWalking tracer out of box, but still you have to install agent or propagate the `sw8` header in your app by yourself.

**Comment by wu-sheng** [comment]
For now, if you want observability but not tracing, check our ALS solution(Envoy access log service solution), which is much better than the old and old performance Istio v1 adaptor solution.

**Comment by zackzhangkai** [comment]
Thanks for your replying. But there are too few docs about how to integrate skywalking to istio for tracing.  https://istio.io/latest/docs/tasks/observability/distributed-tracing/

**Comment by zackzhangkai** [comment]
![image](https://user-images.githubusercontent.com/20178386/124570552-965cb800-de79-11eb-8f48-c1d3ef01cc70.png)


This config takes no effect.

**Comment by wu-sheng** [comment]
@Shikugawa Do you know where Istio update its doc? 

@zackzhangkai We don't care of Zipkin's setting. You are on your own.

**Comment by Shikugawa** [comment]
@zackzhangkai istio 1.10 doesn't have skywalking integration feature. We should wait next release.

**Comment by zackzhangkai** [comment]
@Shikugawa Can you explain it with more details? Do you mean Istio.1.10 not support skywalking for tracing? Thanks for your replying.

**Comment by Shikugawa** [comment]
@zackzhangkai That is merged into upstream. https://github.com/istio/istio/pull/32588#issuecomment-873326966
But it will be included in istio 1.11, not 1.10.

**Comment by wu-sheng** [comment]
> @Shikugawa Can you explain it with more details? Do you mean Istio.1.10 not support skywalking for tracing? Thanks for your replying.

We don't have that support. Tracing in Istio is really very fancy, it just activates tracer in envoy.
